tst/EnergyPlus/unit/Humidifiers.unit.cc

```diff
@@ -0,0 +1,144 @@
+// EnergyPlus::Humidifiers Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// ObjexxFCL Headers
+#include <ObjexxFCL/FArray1D.hh>
+#include <ObjexxFCL/IOFlags.hh>
+#include <ObjexxFCL/gio.hh>
+#include <ObjexxFCL/bit.hh>
```

Comment:
@Myoldmopar.  I was trying to resolve the unit test crash. The solution was was to add Psychometric initialization in the unit test.  If I changed any in Psychometric it must be accidental. Also the "bit.hh" include was done for the same reason. I guess it may not be required, I did not check after the unit test crash issue was resolved.